inbc-program/inbc/parser.py

```diff
@@ -144,11 +118,7 @@ def parse_pota_args(self) -> None:
                                  type=lambda x: validate_string_less_than_n_characters(
```

Comment:
> parser_pota.add_argument('--product', '-pr', default='**kmb-hddl2**', required=False,

Should we remove kmb-hddl2 in pota default value?

Reply:
good point.  I changed everything to what an Alder Lake board returns.